src/platform/user/profile/constants/addressValidationMessages.js

```diff
@@ -10,15 +10,18 @@ export const ADDRESS_VALIDATION_TYPES = Object.freeze({
   VALIDATION_ERROR: 'validationError',
 });
 
+/* eslint-disable jsx-a11y/no-noninteractive-tabindex */
 export const ADDRESS_VALIDATION_MESSAGES = Object.freeze({
   [ADDRESS_VALIDATION_TYPES.BAD_UNIT]: {
     headline: 'Please update your unit number',
     ModalText: ({ editFunction }) => (
       <p>
         We couldn’t verify your address with the U.S. Postal Service because
         there may be a problem with the unit number. Please{' '}
-        <a onClick={editFunction}>edit your address</a> below to update the unit
-        number.
+        <a tabIndex="0" onClick={editFunction}>
```

Comment:
This is fine but I wonder if we could remove the `eslint-disable` by giving this an empty `href`? Or following Jen's recommendation and use a `button` with the `.va-button-link` class applied to it.